src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -195,7 +198,7 @@ - (CGFloat)scrolledPageOffset {
 - (BOOL)isDisplayingPageForIndex:(NSInteger)pageIndex {
   BOOL foundPage = NO;
 
-  // There will never be more than 3 visible pages in this array, so this lookup is
+  // There will never be more than 3 (5 with insets) visible pages in this array, so this lookup is
```

Comment:
The number of visible pages is not constant with a non-zero page inset; it is a function of the page inset, page size, and viewport.